.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!-- Please don't delete this template -->
+
+<!-- PULL REQUEST TEMPLATE -->
+<!-- (Update "[ ]" to "[x]" to check a box) -->
+
+## Summary
+
+*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Close #[issue number]"*
+
+### What are the specific steps to test this change?
+
+*For example:*
+> 1. Run the website and log in as an admin
+> 2. Open a piece manager modal and select several pieces
+> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
+> 4. Check that all pieces have been archived properly
+
+## What kind of change does this PR introduce?
+*(Check at least one)*
+
+- [ ] Bugfix
+- [ ] Feature
+- [ ] Code style update
+- [ ] Refactor
+- [ ] Docs
+- [ ] Build-related changes
+- [ ] Other, please describe:
+
+## Make sure the PR fulfills these requirements:
+
+- [ ] When resolving a specific issue, it references the issue in the PR's title (e.g. `resolves #xxx[,#xxx]` for Github issues, `PRO-xxx` for internal Linear tickets -- where "xxx" is the issue number)
+
+**If there is no related issue/ticket,** the PR's summary includes:
+
+- [ ] A convincing reason for adding this feature or clear description of the bug it resolves
+
+- [ ] The Changelog is updated
+- [ ] Related documentation has been updated
+- [ ] Related tests have been updated
+
+To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.
```

Comment:
Should this read something more like:

```
If adding a new feature, it's best to open a **feature request issue** first and wait for approval before working on it.
```

Alright to leave as is but the way it is currently written has a bit of negative connotation to me.